tests/vm_test.go

```diff
@@ -1891,10 +1886,13 @@ var _ = Describe("[rfe_id:1177][crit:medium][vendor:cnv-qe@redhat.com][level:com
 				Expect(vmRunningRe.FindString(stdout)).ToNot(Equal(""), "VMI is not Running")
 
 				By("Checking no DeletionTimestamp was set")
-				newVMI, err := virtClient.VirtualMachineInstance(vmi.Namespace).Get(vmi.Name, &k8smetav1.GetOptions{})
-				Expect(err).ToNot(HaveOccurred())
-				Expect(newVMI.ObjectMeta.DeletionTimestamp).To(BeNil())
-				Expect(reflect.DeepEqual(newVMI, originalVMI)).To(BeTrue())
+				Consistently(func() error {
+					actualVMI, err := virtClient.VirtualMachineInstance(vmi.Namespace).Get(vmi.Name, &k8smetav1.GetOptions{})
+					Expect(err).ToNot(HaveOccurred())
+					Expect(actualVMI.ObjectMeta.DeletionTimestamp).To(BeNil())
+					return nil
+				}).Should(BeNil())
```

Comment:
Can we define an interval? I think the default one is only 5 seconds.

Reply:
Thanks @xpivarc Absolutely, it was my forgetfulness. Is 30 seconds enough?

Reply:
Yes, that should be more than enough. Can we check what we picked for this kind of test before?
Correction of my statement: It seems the default is only 100 Milliseconds.

Reply:
There is no similar case, but I saw than we made `Eventually` for `300*time.Second, 1*time.Second` when test the `stop` command without dry-run.
Thanks for the correction, I also had to check :slightly_smiling_face: